spec/services/checksum_validator_spec.rb

```diff
@@ -1,7 +1,126 @@
-require 'rails_helper'
+require_relative '../../app/services/checksum_validator.rb'
+require_relative '../load_fixtures_helper.rb'
 
 RSpec.describe ChecksumValidator do
-  it 'has some behaviors' do
-    skip 'we should figure out what they are and test them'
+  let(:storage_dir) { "spec/fixtures/checksum_root01/moab_storage_trunk" }
+  let(:endpoint) { Endpoint.find_by(storage_location: storage_dir) }
+  let(:object_dir) { "#{storage_dir}/#{DruidTools::Druid.new(druid).tree.join('/')}" }
+
+  context '#initialize' do
+    it 'sets attributes' do
+      druid = 'bj102hs9687'
+      cv = described_class.new(druid, storage_dir)
+      expect(cv.druid).to eq "druid:#{druid}"
+      expect(cv.endpoint).to eq endpoint
+      expect(cv.handler_results).to be_an_instance_of AuditResults
+    end
+  end
+
+  context '#validate_manifest_inventories' do
+    let(:druid) { 'bj102hs9687' }
+    let(:cv) { described_class.new(druid, storage_dir) }
+
+    it 'instantiates storage_object from druid and druid_path' do
```

Comment:
same in description for "storage_object" -- use new name or "Moab::StorageObject" in description